jp/lessons/basics/sigils.md

```diff
@@ -1,10 +1,6 @@
 ---
 version: 0.9.0
-layout: page
 title: Sigils

```

Comment:
@gemantzu done in a separate PR https://github.com/doomspork/elixir-school/pull/1122